care/facility/api/serializers/patient.py

```
@@ -210,6 +210,15 @@ def validate_year_of_birth(self, value):
             raise serializers.ValidationError(msg)
         return value
 
+    def validate_name(self, value):
+        if value is None:
+            raise serializers.ValidationError("Patient name is required.")
```

Ruff

Exception must not use a string literal, assign to variable first

Assign to variable; remove string literal

(EM101)